### PR TITLE
Remove headers in forwarder. Add dependencies.

### PR DESCRIPTION
**Pull request by PremyslCerny**

### CODE REVIEW

**Comment by PremyslCerny** [comment]
@ateska Small one.

File: setup.py [hunk 1]
```diff
@@ -62,6 +62,10 @@ def run(self):
 	project_urls={
 		'Source': 'https://github.com/TeskaLabs/asab'
 	},
+	install_requires=[
+		'aiohttp>=3.5.4,<4.0',
+		'fastjsonschema>=2.14.1,<3.0',
```
Comment (ateska) [inline setup.py]: Cool. There is an issue #77 that will be resolved by this.

File: asab/web/authn/oauth/forwarder.py [hunk 2]
```diff
@@ -149,10 +149,9 @@ def __init__(self, *args, container, service, **kwargs):
 		return method
 
 	async def _forward_get(self, request, url, client_id, client_secret):
-		headers = await self._remove_extra_headers(dict(request.headers))
 		data = dict(request.query)
 		async with aiohttp.ClientSession() as session:
-			async with session.get(url, headers=headers, params=data) as resp:
+			async with session.get(url, headers={"Authorization": request.headers.get("Authorization", "")}, params=data) as resp:
```
Comment (ateska) [inline asab/web/authn/oauth/forwarder.py]: The more elegant and future-proof approach is to create` headers` dict prior (just before `data = dict(request.query)` line).

```
data = dict(request.query)
headers = {
}
auth =  request.headers.get("Authorization")
if auth is not None: headers["Authorization"] = auth
    async with aiohttp.ClientSession() as session:
```

File: setup.py (hunk 1 above)
Comment (PremyslCerny) [reply setup.py]: Yep, nice. :)

**Comment by PremyslCerny** [comment]
@ateska Please merge.

**Comment by ateska** [comment]
Fixes #77

**Pull request merged**